include/range/v3/view/any_view.hpp

```diff
@@ -15,6 +15,10 @@
 #ifndef RANGES_V3_VIEW_ANY_VIEW_HPP
 #define RANGES_V3_VIEW_ANY_VIEW_HPP
 
+#include <meta/meta_fwd.hpp>
+
+#if META_HAS_STATIC_RTTI
```

Comment:
Why is this here, vs just avoiding the `#include <range/v3/view/any_view.hpp>` in `<range/v3/view.hpp>`?

Do people want to `#include <any_view.hpp>` directly, even without RTTI, just to declare nothing?

Reply:
I was confused by some test files which just `#include <any_view.hpp>`. I'll change them later.

Reply:
Now `<range/v3/view/any_view.hpp>` and `<range/v3/utility/any.hpp>` are skipped when RTTI is disabled.